### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by PavlovaOlena**
[DEMO LINK](https://PavlovaOlena.github.io/layout_catalog/)
[TEST REPORT LINK](https://PavlovaOlena.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/navigation.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      color: #00acdc;
+    }
+  }
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
```
Comment (witflash) [inline src/styles/blocks/navigation.scss:49]: You can use a nesting for the `after` element:
```
.is-active {
  &::after {
    ...
  }
}
```

File: src/styles/blocks/stars.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  }
+}
+
+.stars {
+  @for $starCount from 1 through 5 {
+    &--#{$starCount} &__star:nth-child(-n + #{$starCount}) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+}
```
[the remark this answers is not in the document]
Comment (witflash) [reply src/styles/blocks/stars.scss:21]: I think 1.a is not correct. How we can use `position` if we need to use a different image?

[the remark this answers is not in the document]
Comment (witflash) [reply src/styles/blocks/stars.scss:21]: About 1.c. We don't use any comments. We have to make a self-describing code style.

**Review by witflash — CHANGES_REQUESTED** [review]
Could you try to fix all tests?

image
You have to add `padding-top` and `padding-bottom` for the `nav__link` and remove `min-height` for the `nav__list`.

File: src/index.html [hunk 3]
```diff
@@ -4,9 +4,330 @@
...
+            src="./images/imac.jpeg" 
+            alt="APPLE A1419 iMac 27"
+          >
+      
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      
+          <p class="card__code">Product code: 195434</p>
+      
+          <div class="card__review">
+            <div class="card__stars stars stars--4">
```
Comment (loralevitska) [inline src/index.html:312]:
```suggestion
            <div class="stars stars--4">
```

File: src/styles/blocks/stars.scss [hunk 4]
```diff
@@ -0,0 +1,21 @@
+.card__stars {
```
Comment (loralevitska) [inline src/styles/blocks/stars.scss:1]: Remove redundant mix `card__stars`
```suggestion
stars {
```

**Review by loralevitska — CHANGES_REQUESTED** [review]
Keep up the good work! Some comments to fix:
- Please fix task point 
image
image

File: src/styles/blocks/card-product.scss [hunk 5]
```diff
@@ -0,0 +1,102 @@
+.main {
+  padding: 50px 40px;
+  box-sizing: border-box;
+}
+
+.container {
```
Comment (anstsot) [inline src/styles/blocks/card-product.scss:6]: it would be great to move this styles to another CSS file

**Review by anstsot — APPROVED** [review]
Great job, approved!

**Pull request closed without merging**